From 9ec6fce3a17a7663ce6923b9a8316bfcb40130bf Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@emb.dev>
Date: Sat, 11 Nov 2023 16:01:37 +0100
Subject: [PATCH] samples: http_get: move 'not CONFIG_NATIVE_LIBC' filter to
 POSIX build

'not CONFIG_NATIVE_LIBC' filter is just to prevent building for
'native_posix' platform with additional options selected by
'sample.net.sockets.http_get.posix' testcase.

Signed-off-by: Marcin Niestroj <m.niestroj@emb.dev>
---
 samples/net/sockets/http_get/sample.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/samples/net/sockets/http_get/sample.yaml b/samples/net/sockets/http_get/sample.yaml
index 999c7ba97d6..2de78ce0bee 100644
--- a/samples/net/sockets/http_get/sample.yaml
+++ b/samples/net/sockets/http_get/sample.yaml
@@ -2,7 +2,7 @@ sample:
   description: BSD Sockets API HTTP GET example
   name: socket_http_get
 common:
-  filter: CONFIG_FULL_LIBC_SUPPORTED and not CONFIG_NATIVE_LIBC
+  filter: CONFIG_FULL_LIBC_SUPPORTED
   harness: net
   min_ram: 32
   min_flash: 80
@@ -16,7 +16,7 @@ tests:
       # Forcibly defines CONFIG_POSIX_API, which is incompatible with
       # CONFIG_NET_SOCKETS_POSIX_NAMES.
   sample.net.sockets.http_get.posix:
-    filter: not CONFIG_NET_SOCKETS_OFFLOAD
+    filter: not CONFIG_NET_SOCKETS_OFFLOAD and not CONFIG_NATIVE_LIBC
     platform_exclude:
       - cc3220sf_launchxl
       - cc3235sf_launchxl
-- 
2.45.2

